### PR TITLE
{lib}[foss/2022a] timm v0.9.2 w/ Python 3.10.4 + CUDA 11.7.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/t/timm/timm-0.9.2-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('PyTorch', '1.12.0', versionsuffix),
+    ('PyYAML', '6.0'),
+    ('tqdm', '4.64.0'),
+    ('torchvision', '0.13.1', versionsuffix),
```
Comment (Micket) [inline easybuild/easyconfigs/t/timm/timm-0.9.2-foss-2022a-CUDA-11.7.0.eb:23]: So, we do have the pytorch bundle now. Should we try to transition to it?

File: easybuild/easyconfigs/t/timm/timm-0.9.2-foss-2022a-CUDA-11.7.0.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+use_pip = True
+
+exts_list = [
+    ('huggingface_hub', '0.15.1', {
+        'checksums': ['a61b7d1a7769fe10119e730277c72ab99d95c48d86a3d6da3e9f3d0f632a4081'],
+    }),
+    ('safetensors', '0.3.1', {
+        'checksums': ['571da56ff8d0bec8ae54923b621cda98d36dcef10feb36fd492c4d0c2cd0e869'],
+    }),
```
Comment (Micket) [inline easybuild/easyconfigs/t/timm/timm-0.9.2-foss-2022a-CUDA-11.7.0.eb:38]: CargoPythonPackage should work here. Even if just defaulting to fetching deps online it's still beneficial to use this easyblock, as it protects your home dir from CARGO_HOME and optimizes march=native

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
We have timm 0.9.7 for this toolchain, using CargoPythonBundle. As this one didn't, it probably no longer even builds as packages have been updated to require newer Rust compilers. Closing this.

**Pull request closed without merging**